### PR TITLE
Add menu option for Always on Top

### LINKED ISSUE #112: Option for chat window staying always on top
Menu or tray menu option to keep the chat windows above others would be nice.

### PR DESCRIPTION
**Pull request by veniversum**
Resolves #112

### CODE REVIEW

File: menu.js [hunk 1]
```diff
@@ -39,6 +40,19 @@ const viewSubmenu = [
 		click() {
 			sendAction('zoom-out');
 		}
+	},
+	{
+		type: 'separator'
+	},
+	{
+		type: 'checkbox',
+		label: 'Always On Top',
```
Comment (sindresorhus) [inline menu.js]: Should be:

> Always on Top

File: menu.js [hunk 2]
```diff
@@ -39,6 +40,19 @@ const viewSubmenu = [
 		click() {
 			sendAction('zoom-out');
 		}
+	},
+	{
+		type: 'separator'
+	},
+	{
```
Comment (sindresorhus) [inline menu.js]: I think it would be more correct to place this in the `Window` menu.

**Comment by veniversum** [comment]
@sindresorhus Updated readme and moved submenu into Window for both OS X and Windows/Linux.

**Comment by sindresorhus** [comment]
I don't have any virtual machines to test on at the moment, but I don't think it's correct to have this in a `Window` menu on Ubuntu and Windows.

**Comment by veniversum** [comment]
@sindresorhus Should it be moved into the `File` or `View` menu for Linux/Windows systems then?

**Comment by sindresorhus** [comment]
Probably `View` then.

**Comment by veniversum** [comment]
Moved it to `View` menu.

File: readme.md [hunk 3]
```diff
@@ -38,6 +38,10 @@ Desktop notifications can be turned on in Preferences.
 
 <div align="center"><img src="media/screenshot-notification.png" width="358"></div>
 
+### Always on Top mode
+
+You can toggle whether Caprine stays on top of other windows in the `Window` menu or with <kbd>Cmd/Ctrl</kbd> <kbd>Shift</kbd> <kbd>t</kbd>.
```
Comment (sindresorhus) [inline readme.md]: Windows/View menu

File: readme.md [hunk 4]
```diff
@@ -51,6 +55,7 @@ Jump to conversation  | <kbd>Cmd/Ctrl</kbd> <kbd>1</kbd>…<kbd>9</kbd>
 Mute conversation     | <kbd>Cmd/Ctrl</kbd> <kbd>Shift</kbd> <kbd>m</kbd>
 Archive conversation  | <kbd>Cmd/Ctrl</kbd> <kbd>Shift</kbd> <kbd>a</kbd>
 Delete conversation   | <kbd>Cmd/Ctrl</kbd> <kbd>Shift</kbd> <kbd>d</kbd>
+Toggle always on top  | <kbd>Cmd/Ctrl</kbd> <kbd>Shift</kbd> <kbd>t</kbd>
```
Comment (sindresorhus) [inline readme.md]: > Toggle "Always on Top"

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**